sdk/search/Azure.Search.Documents/README.md

```diff
@@ -41,7 +40,7 @@ Use the **Azure.Search.Documents client library** to:
 Install the Azure Cognitive Search client library for .NET with [NuGet][nuget]:
 
 ```Powershell
-dotnet add package Azure.Search.Documents --version 1.0.0-preview.4
+dotnet add package Azure.Search.Documents
```

Comment:
Do we want to leave this in until closer to release?  I don't feel super strongly.

Reply:
I wrestled with that myself. How about this: I'll revert that change (though, "preview.5") and add a task list for pre-release activities to the epic.